### DISCUSSION TITLE
Remote references

### DISCUSSION THREAD

**Discussion by rachel-singh** [openapi-processor/openapi-processor-core#69] [General]
Does this support parsing of remote references in the yaml specification?
When I tried it I get an error: Failed to get the schema name: ./components/schemas/Pet.yaml#/components/schemas/Pet
even though the file exists at the specified path and is defined correctly.

**Comment by hauner** [comment]
Theoretically it should work because the processor just passes the api location to the OpenAPI parser. But I never tried it. Your observation suggests that it is not so simple. 

Which parser did you try: SWAGGER, OPENAPI4J, or both?

I will look at it.

**Reply by rachel-singh** [reply]
I tried with OPENAPI4J parser.

Thanks for looking into this.

**Reply by hauner** [reply]
tried it with this simple example: https://raw.githubusercontent.com/openapi-processor/openapi-processor-core/master/src/testInt/resources/tests/ref-into-another-file/inputs/openapi.yaml.

That simple $ref works with both parsers.

Can you provide a link to the OpenAPI files (or an archive with the files) you are trying to process so that I can reproduce the issue?
